database/migrations/2024_08_13_153202_books_make_fields_nullable.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('books', function (Blueprint $table){
+            $table->integer('year')->nullable()->change();
+            $table->string('blurb')->nullable()->change();
+            $table->integer('page_count')->nullable()->change();
+            $table->string('image')->nullable()->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        //
```

Comment:
Usually I would complain about this - but trying to reverse making them nullable might actually not work. So in this rare case, I think it's ok to leave down empty

Reply:
thanks for the feedback ash!